packages/default/scss/common/_forms.scss

```diff
@@ -280,18 +273,14 @@ $input-border-radius: $border-radius !default;
         > .k-numeric-wrap,
         > .k-picker-wrap,
         > .k-multiselect-wrap {
```

Comment:
After those changes, the red invalid border is changed when a widget is hovered/selected. It should be always red.

Reply:
I tweaked the selector slightly, so it doesn't happen